docs/how-to/use-flask-extension.rst

```diff
@@ -0,0 +1,58 @@
+The Flask Extension
```

Comment:
this looks like a good start, it does however mixup reference, explanation, tutorial and how-to 

ref: https://diataxis.fr/

Reply:
Gotcha, I have splited the document into a "how-to" and a "reference".
I also added the `services` field to the extension as well following last week's discussion.